docs/reference/eosmetrics/eosmetrics-docs.xml

```diff
@@ -37,6 +37,11 @@
     <xi:include href="xml/api-index-0.2.xml"><xi:fallback /></xi:include>
   </index>
 
+  <index id="api-index-0.4">
```

Comment:
(This can stay 0.4 though, since the API added in 0.3 is meant for public consumption in 0.4.)